app/controllers/admin/admin_controller.rb

```diff
@@ -3,7 +3,18 @@ class Admin::AdminController < ApplicationController
 
   def authorized?
     if current_user.nil? || !current_user.admin?
-      render plain: "404", status: 404
+      render_404
+    end
+  end
+
+  private
+
+  def render_404
+    respond_to do |format|
+      format.html { render :file => "#{Rails.root}/public/404",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Use the new Ruby 1.9 hash syntax.<br>Block body expression is on the same line as the block start.